argo-cd-apps/overlays/development/repo-overlay.yaml

```diff
@@ -6,8 +6,8 @@ metadata:
     - resources-finalizer.argocd.argoproj.io
 spec:
   source: # This will be replaced with a reference to your fork of this repo (see hack/patch-apps-for-dev.sh)
-    repoURL: https://github.com/redhat-appstudio/infra-deployments.git
-    targetRevision: main
+    repoURL: https://github.com/chrisahl/infra-deployments.git
```

Comment:
hey, just wanted to inform you that we've helper scripts for development purposes, [one](https://github.com/redhat-appstudio/infra-deployments/blob/main/hack/util-update-app-of-apps.sh) of them change the giturl to point to the user's forked repo url and branch. Since, you've hard coded the url to point to your own branch and repo, we could make this a General case.

Reply:
So I had used the development-mode.sh to test out the changes but I forgot that I need to run ./hack/upstream-mode.sh and check the original in before asking for review.  I will fix this.